Experiments/forest_QKS/qks.py

```diff
@@ -0,0 +1,223 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+from sklearn.linear_model import LogisticRegression
+from sklearn.utils import shuffle
+
+from pyquil import get_qc
+from pyquil.quil import Program
+from pyquil.gates import RX, CNOT, MEASURE
+
+
+def make_frames(num):
+    """
+    The picture frames dataset contains a smaller inner square
+    with sides of length 2 and a larger outer square with sides
+    of length 4. Both have points uniformally distributed around
+    the average.
+    """
+    def zip_arrays(d1,d2):
+        return np.array(list(zip(d1,d2)))
+
+    def make_squares(num, l):
+        left_right, top_bottom = make_barriers(num,l), make_barriers(num,l)
+        top_bottom[:,[0,1]] = top_bottom[:,[1,0]] 
+        square = np.concatenate((left_right, top_bottom))
```

Comment:
Your placement of spaces after commas is inconsistent. I recommend that you always put a space after a comma or arithmetic operators.
See: https://stackoverflow.com/questions/18171560/does-pep-8-require-whitespace-around-operators-in-function-arguments